app/services/create_planned_disbursement.rb

```diff
@@ -9,6 +9,7 @@ def call(attributes: {})
     planned_disbursement = PlannedDisbursement.new
 
     planned_disbursement.parent_activity = activity
+    planned_disbursement.report = report(activity: activity)
```

Comment:
I don't think you need to pass the activity as a parameter here, because of the `attr_accessor`.

Reply:
🤦

Reply:
Are we still assuming there will _always_ be an active report available? I'd got the impression that there are times there won't be one (e.g. while the latest one is in review) and we'd need to prevent the creation of transactions and planned disbursements at those times.

Is this taken care of somewhere else, e.g. by blocking access to the form when no active report exists?

Reply:
No this hasn't been taken into account, and I haven't seen any decisions about assuming there will always be an active report available. This logic is in transactions as well (assuming one will always be available). If we need to re-work this then I think we need a separate card to address it for planned disbursements *and* transactions.

Reply:
@james is correct. There will be a period of time when a report is `inactive` (as well as others to come) and we should stop the creation of new entities for this period of time. I hope we can use the policies to do this?

Reply:
James is right, there will be a period of time when a report is inactive (there are others as well) and we should stop entities being created - I am hopeful this can be done via the policies?

There are other states as well, we have the basic mapping of this here: 

https://miro.com/app/board/o9J_knq_KVg=/?moveToWidget=3074457349237052629&cot=12

I don't think this needs to happen as part of this work as we still are in this odd 'one giant backfill report' at the mo.

Reply:
After discussion with @mec I've added a card to address this for PDs and Transactions https://trello.com/c/Yopp4pKf/906-we-cannot-assume-there-will-always-be-an-active-report

Reply:
I'm going to amend this PR to forbid creation of planned disbursements when there is no active report (like Transactions are now) but we'll address the Report's other states in that card I've created.